iwf/workflow_state_options.py

```diff
@@ -13,7 +13,19 @@
 @dataclass
 class WorkflowStateOptions:
     state_id: Optional[str] = None
+    wait_until_api_search_attributes_loading_policy: Optional[
+        PersistenceLoadingPolicy
+    ] = None
+    execute_api_search_attributes_loading_policy: Optional[
+        PersistenceLoadingPolicy
+    ] = None
     search_attributes_loading_policy: Optional[PersistenceLoadingPolicy] = None
+    wait_until_api_data_attributes_loading_policy: Optional[
+        PersistenceLoadingPolicy
+    ] = None
+    execute_api_data_attributes_loading_policy: Optional[
+        PersistenceLoadingPolicy
+    ] = None
```

Comment:
They actually don't look to bad to me at another look, maybe we can just organize them in this order:
```
# apply for both waitUntil and execute API
data_attributes_loading_policy
# apply for both waitUntil and execute API
search_attributes_loading_policy
# below are wait_until API specific options:
wait_until_api_timeout_seconds
wait_until_api_retry_policy
wait_until_api_failure_policy
wait_until_api_data_attributes_loading_policy
wait_until_api_search_attributes_loading_policy
# below are execute API specific options:
execute_api_timeout_seconds
execute_api_retry_policy
execute_api_failure_policy
execute_api_data_attributes_loading_policy
execute_api_search_attributes_loading_policy
```

Reply:
Thanks! This makes sense, done now ✅